charts/spire/charts/spire-server/values.yaml

```diff
@@ -197,3 +197,32 @@ nodeAttestor:
   k8sPsat:
     enabled: true
     serviceAccountAllowList: []
+
+# tornjak - Tornjak specific configuration
+tornjak:
+  enabled: true
+  # image - Tornjak image (frontend + backend) if not separated above
+  image: # ghcr.io/spiffe/tornjak
+    registry: ghcr.io
+    repository: spiffe/tornjak
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    # TODO we should use a specific Tornjak version instead of 'latest'
```

Comment:
+1. What would it take to fix it so that latest isnt needed?

Reply:
Will fix it on the next update. It requires a minor change on Tornjak to be consistent between various Tornjak deployment types.

Reply:
There seems to be work on getting releases here:
https://github.com/spiffe/tornjak/pull/142

Since there is a big warning about the auth feature anyway, I don't think this is a showstopper for merging. We can continue to work on it to get it out of that testing feature mode in follow on pr's.